src/layers/core/scatterplot-layer/scatterplot-layer-fragment.glsl.js

```diff
@@ -35,6 +35,8 @@ void main(void) {
 
   if (distToCenter <= 1.0 && distToCenter >= innerUnitRadius) {
     gl_FragColor = vColor;
+    gl_FragColor = picking_filterHighlightColor(gl_FragColor);
```

Comment:
Is it possible to replace this with
```
if (distToCenter >= 1.0 && distToCenter < innerUnitRadius) {
   discard;
   return;
}
```
It probably won't save any processing but will make the remaining code (which is the "main" path) look cleaner by removing indentation.